src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -840,7 +872,7 @@ var numRowsSelected = 0;
 
 function bindCopyEvents() {
 
-    $('#copyTableModal >tbody>tr').on('click', function(e) {
+    $('body').on('click', '#copyTableModal >tbody>tr', function(e) {
```

Comment:
Similarly, maybe we could use `$('#copyTableModal > tbody > tr').click(function(e) {` here too